.github/workflows/ci.yml

```diff
@@ -0,0 +1,397 @@
+    env:
+      NUM_RUNNERS: ${{ matrix.num_runners }}
+      RUNNER_INDEX: ${{ matrix.runner_index }}
+      TEST_DATABASE_URL: postgres://postgres:postgres@postgres
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      - name: Setup Rust
+        uses: ./.github/actions/setup-rust
+      - name: List packages to test
+        run: |
+          cargo test -p 2>&1 | \
+            awk "/mc-fog/ { print \"-p \" \$1 }" | \
+            grep -v mc-fog-ingest | \
+            sort > /tmp/test-packages
+          split -n l/$RUNNER_INDEX/$NUM_RUNNERS /tmp/test-packages | \
+            tee /tmp/fog-test-packages
+          # Hack: mc-fog-distribution needs bins from
+          # mc-util-{keyfile,generate-sample-ledger}.
+          # TODO: Replace with artifact deps when that does not require
+          # additional cargo flags.
+          grep -q fog-distribution /tmp/fog-test-packages && \
+            echo '-p mc-util-keyfile -p mc-util-generate-sample-ledger' >> /tmp/fog-test-packages || true
+      - name: Run tests
+        uses: ./.github/actions/run-mc-tests
+        with:
+          args: $(cat /tmp/fog-test-packages) --build-jobs 4
+          junit_xml_filename: junit-fog-tests-${{matrix.runner_index}}.xml
+
+    services:
+      # Spin up a PostgreSQL server
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+
+  fog-ingest-tests:
+    runs-on: [self-hosted, Linux, large]
+    container: mobilecoin/rust-sgx-base:latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      - name: Setup Rust
+        uses: ./.github/actions/setup-rust
+      - name: Run tests
+        uses: ./.github/actions/run-mc-tests
+        with:
+          # These tests time out without release mode.
+          args: -p 'mc-fog-ingest-*' --release
+          junit_xml_filename: junit-fog-ingest-tests.xml
+    env:
+      TEST_DATABASE_URL: postgres://postgres:postgres@postgres
+
+    services:
+      # Spin up a PostgreSQL server
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+
+  fog-conformance-tests:
+    runs-on: [self-hosted, Linux, large]
+    container: mobilecoin/rust-sgx-base:latest
+
+    env:
+      PGHOST: postgres
+      PGUSER: postgres
+      PGPASSWORD: postgres
+      DATABASE_URL: postgres://postgres:postgres@postgres/fog
```

Comment:
why does this and the one below use `@postgres/fog` while the other fog tests use `@postgres`?